holoviews/core/util.py

```diff
@@ -929,7 +929,7 @@ def isfinite(val):
     if pandas_version >= Version('1.0.0'):
         if finite is pd.NA:
             return False
-        return finite & (~pd.isna(val))
+        return finite & ~pd.isna(np.asarray(val))
```

Comment:
From this Python 3.12 warning: ` Bitwise inversion '~' on bool is deprecated. This returns the bitwise inversion of the underlying int object and is usually not what you expect from negating a bool. Use the 'not' operator for boolean negation or ~int(x) if you really want the bitwise inversion of the underlying int.`

And val sometimes could be None or a tuple of None. 
![ Screenshot 2023-10-10 13 46 46](https://github.com/holoviz/holoviews/assets/19758978/2fd911ed-800c-48dd-8d21-3dcd0c986e9c)